pkg/apiservers/openshiftapiserver.go

```diff
@@ -0,0 +1,225 @@
+package apiservers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	operatorV1 "github.com/openshift/api/operator/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var openshiftAPIServerObjName = "cluster"
+
+// OpenshiftAPIServerBuilder provides struct for openshiftAPIServer object.
+type OpenshiftAPIServerBuilder struct {
+	// OpenshiftAPIServer definition. Used to create an openshiftAPIServer object.
+	Definition *operatorV1.OpenShiftAPIServer
+	// Created openshiftAPIServer object.
+	Object *operatorV1.OpenShiftAPIServer
+	// apiClient opens api connection to the cluster.
+	apiClient *clients.Settings
```

Comment:
good point. But I can do it for the openshiftapiserver only. In case of kubeapiserver I have missing operations issues. Like I can not Get an object for example.

It will be kind of inconsistency between this two (and, I believe, it can be more added)
IMHO, so only if you insist I can change it for the openshiftapiserver.

Reply:
@kononovn WDYT?

Reply:
we've back to the inconsistency.
And why we need two variables?
@kononovn @trewest WDYT?

Reply:
I'll wait for any response/advice from @kononovn or @trewest before I do any change if you don't mind

Reply:
change was done according to the @trewest recomentations